src/ngrx/8-test-api-effects/test-api-effects.md

```diff
@@ -5,4 +5,31 @@
 
 @body
 
-TODO
\ No newline at end of file
+> **Quick Start**: You can checkout [this branch](https://github.com/bitovi/angular-ngrx-chat/tree/create-api-effects) to get your codebase ready to work on this part.
+
+## Running Tests
+
+To run unit tests in your project, you can either use the `test` npm script, or the `ng test` command:
+
+```bash
+npm run test
+# or
+ng test --watch
+```
+
+The `--watch` switch will rerun your tests whenever a code file changes. You can skip it to just run all tests once.
+
+## Test Code
+
+You'll need to copy the contents of the test file to run tests for your Effects.
+
+### Update `login.effects.spec.ts` 
+
+Copy the following code to replace the contents of `src/app/store/login/login.effects.spec.ts`:
+
+<details>
+<summary>src/app/store/login/login.effects.spec.ts</summary>
+@sourceref ./login.effects.spec.ts
```

Comment:
this file is the same content as [/10-test-redirect-effects/test-redirect-effects.md](https://github.com/bitovi/academy/pull/339/files#diff-74819210d074e958921d3b197d86c614ce2a0f8cd779998e2c324446ae987bdd), is this correct?

Reply:
(except for quick start and wrap up)

Reply:
I'll resolve this conversation. I'm assuming what changes is the content of the login.effects.spec.ts, as it is probably the same folder structure